File: PHP/Cap04/c04ex05.php
<?php
// c04ex05.php

printf("CONVERTER NUMERO DO MES PARA O NOME DO MES\n\n");

printf("Entre o numero do certo mes: ");
$Mes = intval(trim(fgets(STDIN)));

switch ($Mes) {
  case 1:
    printf("Janeiro\n");
    break;
  case 2:
    printf("Fevereiro\n");
    break;
  case 3:
    printf("Março\n");
    break;
  case 4:
    printf("Abril\n");
    break;
  case 5:
    printf("Maio\n");
    break;
  case 6:
    printf("Junho\n");
    break;
  case 7:
    printf("Julho\n");
    break;
  case 8:
    printf("Agosto\n");
    break;
  case 9:
    printf("Setembro\n");
    break;
  case 10:
    printf("Outubro\n");
    break;
  case 11:
    printf("Novembro\n");
    break;
  case 12:
    printf("Dezembro\n");
    break;
  default:
    printf("Mes invalido\n");
    break;
}

printf("\n");
printf("Tecle <Enter> para encerrar... ");
trim(fgets(STDIN));

?>

File: SCALA/Cap04/c04ex05.scala
// c04ex05.scala

import scala.io.StdIn._

object Main {
  def main(args: Array[String]): Unit = {
    var Mes: Int = 0

    print("Entre com um numero equivalente a um mes: ")
    Mes = readInt()

    Mes match {
      case  1 => println("Janeiro")
      case  2 => println("Fevereiro")
      case  3 => println("Março")
      case  4 => println("Abril")
      case  5 => println("Maio")
      case  6 => println("Junho")
      case  7 => println("Julho")
      case  8 => println("Agosto")
      case  9 => println("Setembro")
      case 10 => println("Outubro")
      case 11 => println("Novembro")
      case 12 => println("Dezembro")
      case  _ => println("Mês inválido")
    }

    print("\nTecle <Enter> para encerrar... ")
    readLine()
  }
}
